swarm/network_test.go

```diff
@@ -530,3 +532,31 @@ func retrieve(
 
 	return uint64(totalCheckCount) - atomic.LoadUint64(totalFoundCount)
 }
+
+// putString provides singleton manifest creation on top of api.API
```

Comment:
is this not the same `putString` as above?

Reply:
it is, but i don't want to have this function inside the `API` object. also, putting it into `testutil` results in cyclic dependency. I'll try to find a better place for it :+1:

Reply:
so @zelig, I had a look at this. Few things:
1. why is `network_test.go` in this directory?
2. do we really need these tests? who runs them and why? seems they are all long-running and pretty much black-box
3. we should not have anything calling `api` directly here. it is another manifestation of our malformed testing practices. we should be reverting to testing at borders as @nonsense suggested. the fact that this test calls API directly shows there's a problem here. we should be using the http api instead. I'm gonna leave this as is for the moment because this is out of scope for this PR but I'm gonna take care of this.

Reply:
see #1364